steps/pcloudy-app-upload/0.1.2/assets/icon.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 256 257"><defs><style>.cls-1{fill:#fff;}.cls-2{fill:#07223e;}</style></defs><title>Asset 1</title><g id="Layer_2" data-name="Layer 2"><g id="Layer_1-2" data-name="Layer 1"><rect class="cls-1" width="256" height="257"/><g id="Layer_1-2-2" data-name="Layer 1-2"><path class="cls-2" d="M190.61,111.92a56.82,56.82,0,0,0-66.68-44.85q-2.4.47-4.74,1.14a35.3,35.3,0,0,1,10.09,8.18,42.85,42.85,0,0,1,7.22,13.38A49.26,49.26,0,0,1,139,105.24a53,53,0,0,1-4.58,21.17A64.71,64.71,0,0,1,122,145.21a65.72,65.72,0,0,1-18.18,13.47A48.21,48.21,0,0,1,81.83,164H76.65l-4.4,20.37a45.36,45.36,0,0,0,16.9,3.33h98.66c17.53,0,33.16-11.87,37.1-29a37.87,37.87,0,0,0-27.79-45.77A38.81,38.81,0,0,0,190.61,111.92Z"/><path class="cls-2" d="M64.1,151.5H84.32A36.26,36.26,0,0,0,101,147.55a49.41,49.41,0,0,0,13.7-10.18,48.53,48.53,0,0,0,9.4-14.23,39.68,39.68,0,0,0,3.46-16,36.6,36.6,0,0,0-1.89-11.69,32,32,0,0,0-5.44-10.11,27.28,27.28,0,0,0-8.72-7.11A25.22,25.22,0,0,0,99.8,75.53H54.6L30.77,187.62H56.53l1.09-5.12ZM75.51,99.2H93a7,7,0,0,1,5.39,2.61c1.55,1.76,2.31,4.47,2.31,8.17a25.33,25.33,0,0,1-1.14,7.93,20.36,20.36,0,0,1-3.1,6,13.91,13.91,0,0,1-4.3,3.85A9.83,9.83,0,0,1,87.49,129H69.2Z"/></g></g></g></svg>
```

Comment:
assets dir should go to the step group dir (`steps/pcloudy-app-upload/assets/`), not under a specific step version, example: https://github.com/bitrise-io/bitrise-steplib/tree/master/steps/activate-ssh-key